core/.changelog.d/3868.added

```diff
@@ -0,0 +1,2 @@
+Added flag for setting up device using SLIP39 "single".
+Renamed flag used for setting up device using BIP39 to `basic`.
```

Comment:
these should be two changelog entries, one for `added`, other for `incompatible`